_dev/src/ts/components/ModalContainer.ts

```diff
@@ -0,0 +1,57 @@
+import Hydration from '../utils/Hydration';
+
+export default class ModalContainer {
+  public static readonly cancelEvent = 'cancel';
+  public static readonly okEvent = 'ok';
+
+  public static readonly containerId = 'ua_modal';
+
+  public init(): void {
+    this.modalContainer.addEventListener(Hydration.hydrationEventName, this.displayModal);
+    this.modalContainer.addEventListener('click', this.onClick);
+    this.modalContainer.addEventListener(ModalContainer.cancelEvent, this.closeModal);
+    this.modalContainer.addEventListener(ModalContainer.okEvent, this.closeModal);
+  }
+
+  public unload(): void {
```

Comment:
see previous comment
```suggestion
  public beforeDestroy(): void {
```